e2e/e2e_metric_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_test
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	// metrics obtained from the kepler exporter:
+	// curl http://localhost:9102/metrics |awk -F"\{" '{print $1}' | grep kepler_ |grep -v \# |sort  |uniq  |xargs -I {} echo \"{}\",
+	kepler_metrics = []string{
+		"kepler_container_core_joules_total",
+		"kepler_container_cpu_cpu_time_us",
+		"kepler_container_dram_joules_total",
+		"kepler_container_gpu_joules_total",
+		"kepler_container_joules_total",
+		"kepler_container_other_host_components_joules_total",
+		"kepler_container_package_joules_total",
+		"kepler_container_uncore_joules_total",
+		"kepler_exporter_build_info",
+		"kepler_node_core_joules_total",
+		"kepler_node_cpu_scaling_frequency_hertz",
```

Comment:
"kepler_container_core_joules_total"
"kepler_container_dram_joules_total"
"kepler_container_gpu_joules_total"
"kepler_container_joules_total"
"kepler_container_other_host_components_joules_total"
"kepler_container_package_joules_total"
"kepler_container_uncore_joules_total"
"kepler_exporter_build_info"
"kepler_node_energy_stat"
"kepler_node_nodeInfo"
"kepler_node_other_host_components_joules_total"
"kepler_node_platform_joules_total"
"kepler_pod_energy_stat"


from my local VM env
something like `kepler_node_dram_joules_total` (not only this but should be simliar) are missing

Reply:
I am guessing 

https://github.com/sustainable-computing-io/kepler/blob/main/pkg/collector/node_energy_collector.go#L47
return empty nodeComponentsEnergy due to both conditions above not satisifed ..

Reply:
see above ,that's my output ...

Reply:
@rootfs 

this might because the conf by default not provided so that the LinearRegressor init failed for node ...
and in turn https://github.com/sustainable-computing-io/kepler/blob/main/pkg/collector/node_energy_collector.go#L47
returns empty
```
I1116 02:24:17.933746       1 lr.go:166] LR Model (AbsModelWeight): no config
I1116 02:24:17.933751       1 lr.go:166] LR Model (AbsComponentModelWeight): no config
I1116 02:24:17.933755       1 lr.go:166] LR Model (DynModelWeight): no config
I1116 02:24:17.933758       1 lr.go:157] getWeightFromServer failed, using InitModelURL:https://raw.githubusercontent.com/sustainable-computing-io/kepler-model-server/main/tests/test_models/DynComponentModelWeight/CgroupOnly/ScikitMixed/ScikitMixed.json to load
```